CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Contributors Guide
+
+ If you believe that you have found a bug, please search for an existing [issue](https://issues.apache.org/jira/browse/accumulo) to see if it has already been reported. If you would like to add a new feature to Accumulo, please send an email with your idea to the [user](mailto:user@accumulo.apache.org) mail list. If it's appropriate, then we will create a ticket and assign it to you.
+
+## Development
+
+- Ensure that your work targets the correct branch
+- Add / update unit and integration tests
+- Ensure that Accumulo builds cleanly using the command: `mvn clean verify`
```

Comment:
I think the intent is that `mvn verify` passes before the change is submitted, but this reads as those devs should always use that (when in practice, `mvn package` might be just fine).